x-pack/plugins/enterprise_search/public/applications/shared/not_found/not_found_prompt.tsx

```diff
@@ -47,9 +52,10 @@ export const NotFoundPrompt: React.FC<Props> = ({ productSupportUrl, backToLink
         <EuiFlexGroup>
           <EuiFlexItem>
             <EuiButtonTo to={backToLink} fill>
-              {i18n.translate('xpack.enterpriseSearch.notFound.action1', {
-                defaultMessage: 'Back to your dashboard',
-              })}
+              {backToContent ??
+                i18n.translate('xpack.enterpriseSearch.notFound.action1', {
+                  defaultMessage: 'Back to your dashboard',
```

Comment:
Nit: Is there better wording that we could use here? Even `Back to dashboard`?

Reply:
This was existing so I just left the existing copy.